homeassistant/components/tellduslive/entry.py

```diff
@@ -0,0 +1,115 @@
+"""Base Entity for all TelldusLiveEntities."""
+from datetime import datetime
+import logging
+
+from homeassistant.const import ATTR_BATTERY_LEVEL, DEVICE_DEFAULT_NAME
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, SIGNAL_UPDATE_ENTITY
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_LAST_UPDATED = 'time_last_updated'
+
+
+class TelldusLiveEntity(Entity):
+    """Base class for all Telldus Live entities."""
+
+    def __init__(self, hass, device_id):
+        """Initialize the entity."""
+        self._id = device_id
+        self._client = hass.data[DOMAIN]
+        self._name = self.device.name
+        self._async_unsub_dispatcher_connect = None
+        _LOGGER.debug('Created device %s', self)
```

Comment:
Please remove or move this to outside the entity. We don't want side effects like logging in the constructor.

Reply:
Just remove. It's a duplicate since we log in `async_added_to_hass` below.